Makefile

```diff
@@ -128,14 +146,13 @@ tag-cli-release-image: push-cli-image
 	true
 
 .PHONY: commitlint
-commitlint:
-	gh pr view $${GITHUB_HEAD_REF} --json title,body --template '{{.title}}{{ "\n\n" }}{{.body}}' > commitlint.msg
-	@echo "commit message that will be linted:"
-	@cat commitlint.msg
-	@echo
-	earthly +commitlint
+commitlint: commitlint.msg
```

Comment:
please introduce a variable for `commitlint.msg`, it is used in a couple places.